### ISSUE TITLE
Piskel zooms in&out way too fast on firefox

### ISSUE THREAD

**Issue by SABITRON** [piskelapp/piskel#1079]
When using firefox, scrolling in and out on the canvas zooms the view way too close or way too far, with no possible way to fine-tune the zoom to properly edit a file of any resolution

**Comment by ayushmxn** [comment]
If you pinch in and out for zoom it is pretty fast. I think that is by design because if you drag two fingers up/down the keypad, it does a very fine grained slow zoom. I'm using a laptop running Windows. Not sure how this would translate for desktops though.

**Comment by giusgad** [comment]
@SABITRON You can use the `+` and `-` keys to zoom more precisely, or `0` to reset the zoom level.

**Comment by bannmannutt** [comment]
This is still a problem. Strangely enough, if I switch my user agent to chrome, while staying in the firefox browser, zooming is entirely disabled, but switching my user agent to safari causes zooming to act totally normally.

**Comment by Goon1170** [comment]
Bumping this. STILL a problem, only on Firefox

**Comment by enykwest** [comment]
Yeah... I used the online version to decide if I wanted to get into pixel art, but zooming was ROUGH so I downloaded the offline version which is much nicer. I pushed through the pain, but I suspect there are others who simply gave up and moved onto something else.
